packages/hardware-trezor/src/transformers/tx.ts

```diff
@@ -34,22 +34,23 @@ const trezorTxTransformer = async (
   };
 };
 
-/**
- * Temporary props type extending the existing TxToTrezorProps
- * and adding the core Cardano.TxBody so we can pass it to our
- * override implementations.
- */
-type TxToTrezorProps = deprecatedUtil.TxToTrezorProps & {
-  cardanoTxBody: Cardano.TxBody;
-};
-
 /**
  * Takes a core transaction and context data necessary to transform
  * it into a trezor.CardanoSignTransaction
  */
-export const txToTrezor = (props: TxToTrezorProps): Promise<Omit<Trezor.CardanoSignTransaction, 'signingMode'>> =>
-  trezorTxTransformer(props.cardanoTxBody, {
-    chainId: props.chainId,
-    inputResolver: props.inputResolver,
-    knownAddresses: props.knownAddresses
+export const txToTrezor = ({
+  cardanoTxBody,
+  chainId,
+  inputResolver,
+  knownAddresses
+}: {
+  chainId: Cardano.ChainId;
+  inputResolver: Cardano.InputResolver;
+  knownAddresses: GroupedAddress[];
+  cardanoTxBody: Cardano.TxBody;
+}): Promise<Omit<Trezor.CardanoSignTransaction, 'signingMode'>> =>
+  trezorTxTransformer(cardanoTxBody, {
+    chainId,
+    inputResolver,
+    knownAddresses
```

Comment:
I used this from https://github.com/input-output-hk/cardano-js-sdk/pull/903 to avoid additional work once both PR's are merged